extensions/amp-font/amp-font.md

```diff
@@ -26,9 +26,16 @@ limitations under the License.
 
 # amp-font
 
-## Behavior
+## Do not use
+
+Use the [`font-display`](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/font-display) descriptor to specify how to display a font face dependent on whether and when it is ready. 
+
+The `amp-font` component was AMP's solution to controlling timeouts on font loading. With the wide browser adoption of font-display, this component is no longer necessary. It is here for legacy users. 
+
+### Usage
```

Comment:
Suggestion: remove the old documentation.

Reply:
I'm torn about removing it completely. 137 million pages might still need it...